index.html

```diff
@@ -163,12 +239,35 @@
       .attr('r', 5);
 
     points
-      .attr('cx', function (d) { return x(d.unemployment); })
-      .attr('cy', function (d) { return y(d.inflation); });
+      .attr('cx', function (d) { return x(d.year); })
+      .attr('cy', function (d) { return y(d.value); });
 
     points.exit().remove();
 
     // HISTOGRAMS
+    var hist = d3.layout.histogram()
+       .value(function (d) { return d.year; })
+       .bins(x.ticks(binCount)
```

Comment:
Have you defined binCount in this file anywhere? Note that the histogram layout is trying to reference that.